### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by vbkaisetsu**
This PR adds the Code of Conduct of daac-tools community.
This guideline is modified from the Ubuntu Code of Conduct.

### CODE REVIEW

File: conduct.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+account when making decisions. Even if it's not obvious at the time, our contributions will impact
+the work of others. For example, changes to code, infrastructure, policy, documentation, and
+translations during a release may negatively impact others' work.
+
+Be respectful.
+--------------
+
+The daac-tools community and members treat one another with respect. Everyone can make a valuable
+contribution to daac-tools. We may not always agree, but disagreement is no excuse for poor
+behavior and poor manners. We might all experience some frustration now and then, but we cannot
```
Comment (kampersanda) [inline conduct.md]:
```suggestion
behavior and manners. We might all experience some frustration now and then, but we cannot
```

File: conduct.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+exception. It is important that we resolve disagreements and differing views constructively and
+with the help of the community and community processes. We have places available to facilitate
+discussion, and these places help to decide the right course for daac-tools. When our goals differ
+dramatically, we encourage the creation of alternative projects so that the community can test new
+ideas and contribute to the discussion.
+
+When we are unsure, we ask for help.
+------------------------------------
+
+Nobody knows everything, and nobody is expected to be perfect in the daac-tools community. Asking
```
Comment (kampersanda) [inline conduct.md]:
```suggestion
Nobody knows everything and is expected to be perfect in the daac-tools community. Asking
```

File: conduct.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+questions avoids many problems down the road, so questions are encouraged. Those who are asked
+questions should be responsive and helpful. However, when asking a question, care must be taken to
+do so in an appropriate forum.
```
Comment (kampersanda) [inline conduct.md]:
```suggestion
questions should be responsive and helpful. However, when asking a question, care must be taken
in an appropriate forum.
```

**Comment by vbkaisetsu** [comment]
@kampersanda I used the Ubuntu Code of Conduct v1.1 in the previous commit, but I thought v2.0 is better to reuse, so I changed the sentences. Could you review again?

**Review by odashi — APPROVED** [review]

**Review by kg86 — APPROVED** [review]

**Review by kampersanda — APPROVED** [review]

**Pull request merged**